pkg/kube/ready.go

```diff
@@ -90,13 +90,6 @@ type ReadyChecker struct {
 // IsReady will fetch the latest state of the object from the server prior to
 // performing readiness checks, and it will return any error encountered.
 func (c *ReadyChecker) IsReady(ctx context.Context, v *resource.Info) (bool, error) {
-	var (
-		// This defaults to true, otherwise we get to a point where
-		// things will always return false unless one of the objects
-		// that manages pods has been hit
-		ok  = true
-		err error
-	)
```

Comment:
Looking over 10920 I get why we need the rest of the fixes, but can you help clarify this part? Tempted to rubber stamp but just checking on this.

Reply:
Those variables were used as part of the code that was removed. They are now unused variables so the Go compiler gives an area. The code that uses them is gone.